core/remoteviewserver.cpp

```diff
@@ -91,7 +93,6 @@ bool RemoteViewServer::isActive() const
 
 void RemoteViewServer::sendFrame(const RemoteViewFrame &frame)
 {
-    m_clientReady = false;
```

Comment:
After writing all that it make it clear that the problem comes from the differences between the different grabbers... widget grabber is synchrone while the quick one is asynchrone now (used to be synchrone before but no longer). So we did not get that corner case before.